packages/scripts/config/webpack.development.js

```diff
@@ -26,6 +27,8 @@ module.exports = merge(webpackConfig('development'), {
     },
     inline: true,
     overlay: true,
+    // checks for override in redwood.toml, defaults to true
+    open: !redwoodConfig.browser || redwoodConfig.browser.open,
```

Comment:
You could use the new optional conditional operator:
`redwoodConfig?.browser?.open || true`

Reply:
zOMG Where has this been all my life?!? 

I need to beef up my knowledge of current js and ts. Please keep these coming. And I’ll keep reading the docs.